sw/device/silicon_creator/rom/keys/real/rsa/hsmtool/earlgrey_keygen.hjson

```diff
@@ -0,0 +1,259 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// The earlgrey keys were generated by the following command:
+// hsmtool -t <token> -u user -p <pin> exec earlgrey_keygen.hjson
+
+[
+  ////////////////////////////////////////////////////////////
+  //
+  // There are 3 prod keys named earlgrey_a0_prod_{0,1,2}
+  //
+  ////////////////////////////////////////////////////////////
+  {
+    command: "rsa-generate",
+    label: "earlgrey_a0_prod_0",
+    key_length: 3072,
+    wrapping: false,
+    extractable: false,
+    public_exponent: 65537,
+    public_template: {
+          CKA_CLASS: "CKO_PUBLIC_KEY",
+          CKA_KEY_TYPE: "CKK_RSA",
+          CKA_TOKEN: true,
+          CKA_ENCRYPT: false,
+          CKA_VERIFY: true
+    },
+    private_template: { 
+          CKA_CLASS: "CKO_PRIVATE_KEY",
+          CKA_KEY_TYPE: "CKK_RSA",
+          CKA_TOKEN: true,
+          // The key is valid for signing, but not decryption or unwrapping.
+          CKA_SIGN: true
+          CKA_DECRYPT: false,
+          CKA_UNWRAP: false,
+          // The key is not extractable from the HSM.
+          CKA_EXTRACTABLE: false,
+          // The key may not be accessed until the user has authenticated
+          // to the HSM.
+          CKA_PRIVATE: true,
+          // The key may not be revealed in plaintext outside the HSM.
+          CKA_SENSITIVE: true,
+	  CKA_MODIFIABLE: true,
+	  CKA_DESTROYABLE: false,
```

Comment:
I think this is OK for this PR but we actually want this the other way around
```suggestion
	  CKA_MODIFIABLE: false,
	  CKA_DESTROYABLE: true,
```
don't we? E.g. we don't want to be able to create the keys and set `EXTRACTABLE` to `true`. If some of these attributes only support a one way transition we should document those.

Reply:
The current settings (modifiable=true and destroyable=false) are what was agreed upon in the documentation.

I we want `destroyable=false` to prevent an accidental key deletion.
I don't think we can change `extractable` from false to true after creating keys.  I agree this needs investigation and documentation.